packages/fiori3/src/components/FilterBar/index.tsx

```diff
@@ -38,11 +36,7 @@ export class FilterBar extends PureComponent<FilterBarPropTypes> {
       <div className={classes.outerContainer}>
         <div className={classes.filterBarHeader}>
           {renderVariants && renderVariants()}
-          {renderSearch && (
-            <div className={classes.vLine}>
-              <Input placeholder={'Search'} />
-            </div>
-          )}
+          {renderSearch && <div className={classes.vLine}> {renderSearch()} </div>}
```

Comment:
Should we check if the passed prop is a function? In case a React Element is passed, this will crash in my opinion.
What do you think?